grail/steps.py

```diff
@@ -142,7 +142,7 @@ def print_to_console():
 
 
 def step(func=None, description='', pending=False, step_group=False, format_description=False,
-         treat_nested_steps_as_methods=False, log_output=True, log_input=True):
+         treat_nested_steps_as_methods=False, log_output=True, log_input=True, clean_params=True):
```

Comment:
If you don't need such global switch, let's skip this proposal. Maybe `clean_params=True` should be renamed to `use_self_in_args=False`?